backend/src/main/java/codesquard/app/label/dto/LabelSaveResponse.java

```diff
@@ -1,16 +1,16 @@
 package codesquard.app.label.dto;
 
-public class LabelSavedResponse {
+public class LabelSaveResponse {
 	private final boolean success;
 	private final Long id;
 
-	private LabelSavedResponse(boolean success, Long id) {
+	private LabelSaveResponse(boolean success, Long id) {
 		this.success = success;
 		this.id = id;
 	}
 
-	public static LabelSavedResponse success(boolean success, Long id) {
-		return new LabelSavedResponse(success, id);
+	public static LabelSaveResponse success(boolean success, Long id) {
+		return new LabelSaveResponse(success, id);
```

Comment:
```suggestion
	public static LabelSaveResponse success(Long id) {
		return new LabelSaveResponse(true, id);
```
팩토리 메소드의 용도가 성공시 응답하는 것이기 때문에 별도의 boolean 매개변수를 받지 않고 객체 생성시 true를 전달하면 좋을것 같습니다.

Reply:
감사합니다! 리팩토링할 때 전체적으로 말씀하신 것 적용하겠습니다